packages/neo4j-driver/src/session-rx.js

```diff
@@ -104,6 +107,45 @@ export default class RxSession {
     return this._runTransaction(ACCESS_MODE_WRITE, work, transactionConfig)
   }
 
+
+  /**
+   * Executes the provided unit of work in a {@link READ} reactive transaction which is created with the provided
+   * transaction configuration.
+   * @public
+   * @param {function(txc: RxManagedTransaction): Observable} work - A unit of work to be executed.
+   * @param {TransactionConfig} transactionConfig - Configuration for the enclosing transaction created by the driver.
+   * @returns {Observable} - A reactive stream returned by the unit of work.
+   */
+   executeRead (work, transactionConfig) {
```

Comment:
```suggestion
   executeRead(work, transactionConfig) {
```
nit: is the space put on purpose here? (same question below)

Reply:
JS Standard formatting requires a space after the method name. It is ensured by linter/autoformatter.